src/validation/validators/fileType/FileTypeValidation.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { FileTypeValidation } from './FileTypeValidation'
+import { InvalidFileTypeError } from '@/validation/errors'
+import faker from 'faker'
+
+const allowedFileExtensions = ['png', 'jpg', 'jpeg', 'pdf', 'txt'];
+
+const makeSut = (fieldName: string): FileTypeValidation =>
+  new FileTypeValidation(fieldName, allowedFileExtensions)
+
+describe('FileTypeValidation', () => {
+  test('Should return error if file extension is not allowed', () => {
+    const field = faker.database.column();
```

Comment:
you could put the ` const field = faker.database.column();` in line 6, behind allowedFileExtensions, because you repeat it in all tests.